pkg/agent/datamodel/azenvtypes.go

```diff
@@ -273,12 +273,4 @@ var (
 			AKSUbuntu1804:  AKSUbuntu1804OSImageConfig,
 		},
 	}
-
-	// AzureCloudSpecEnvMap is the environment configuration map for all the Azure cloud environments.
-	AzureCloudSpecEnvMap = map[string]AzureEnvironmentSpecConfig{
-		AzureChinaCloud:        AzureChinaCloudSpec,
```

Comment:
Should we also remove these Specs? i.e. AzureChinaCloudSpec, AzureGermanCloudSpec...

Reply:
You are right, we don't need them. Just removed.